datastream-cloud-storage/src/main/java/com/linkedin/datastream/cloud/storage/io/ScribeParquetFile.java

```diff
@@ -0,0 +1,92 @@
+/**
+ *  Copyright 2020 Wayfair LLC. All rights reserved.
+ *  Licensed under the BSD 2-Clause License. See the LICENSE file in the project root for license information.
+ *  See the NOTICE file in the project root for additional information regarding copyright ownership.
+ */
+package com.linkedin.datastream.cloud.storage.io;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.hadoop.fs.Path;
+import org.apache.parquet.avro.AvroParquetWriter;
+import org.apache.parquet.hadoop.ParquetWriter;
+import org.apache.parquet.hadoop.metadata.CompressionCodecName;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linkedin.datastream.cloud.storage.Package;
+import com.linkedin.datastream.common.VerifiableProperties;
+
+/**
+ * Implementation of {@link com.linkedin.datastream.cloud.storage.io.File} to support parquet file format for scribe
+ */
+public class ScribeParquetFile implements com.linkedin.datastream.cloud.storage.io.File {
```

Comment:
I'd rename this class to `ScribeRawParquetWriter` since the schema is specific to raw JSON data. We will soon need a writer to handle Scribe Event Parquet files that have schemas stored in SchemaRegistry.